sklearn/metrics/cluster/_unsupervised.py

```diff
@@ -109,6 +109,16 @@ def silhouette_score(
     silhouette : float
         Mean Silhouette Coefficient for all samples.
 
+    Examples
```

Comment:
The CI fails because the examples should be the last section of the docstring by the numpydoc convention. Can you move it to the end.

Reply:
Committed the new changes